src/main/java/net/runelite/data/dump/MediaWikiTemplate.java

```diff
@@ -99,8 +101,11 @@
 	@Nullable
 	public static MediaWikiTemplate parseWikitext(final String name, final String data)
 	{
+		final Pattern exactNameTest = Pattern.compile(name.toLowerCase() + "\\s*\\|");
```

Comment:
you can just make the pattern case-insensitive. Also, if you want to match | also match {{ at start